### ISSUE TITLE
Ollama JSON parse error

### ISSUE THREAD

**Issue by defaultsecurity** [micz/ThunderAI#142]
### The problem

During trannsating a message, half-way througI got this error:
```
Ollama API request failed: JSON.parse: unterminated string at line 1 column 80 of the JSON data
```

I think it could be an escaping problem.

### Which Operative System are you using?

Windows 11

### Which version of Thunderbird are you using?

128.2.3esr

### Which version of ThunderAI has the issue?

2.1.5

### Which integration are you using?

Ollama API

### Anything in the Thunderbird console logs that might be useful?

```txt
Ollama API request failed: JSON.parse: unterminated string at line 1 column 80 of the JSON data
```

### Additional information

Ollama version 0.3.12

**Comment by micz** [comment]
May you activate the debug and post here the error log?
May you also provide a prompt to replicate the error?
Thank you.

**Comment by micz** [comment]
I'm closing this one, please refer to #145.

**Comment by micz** [comment]
The problem will be fixes in 2.2.0, see #147.

**Comment by micz** [comment]
I've released version 2.2.0pre5, if you want to test it.
See the changelog: https://github.com/micz/ThunderAI/releases/tag/v2.2.0pre5
Thank you.

**Comment by micz** [comment]
Fixed in version 2.2.0.
